user-guide/Getting_started/The_DataMiner_User_Interface/Monitoring_app/Card_Pane/Alarm_Cards.md

```diff
@@ -21,4 +21,7 @@ In the header of the card, two buttons are available:
 - *Ownership*: Allows you to take or release ownership of the alarm.
 
 Alarm cards can be accessed from the Alarm Console, but you can also navigate directly to an alarm card by specifying the following URL:
-<br>*http://*\<DMA IP>*/monitoring/alarm/*\<DMA ID>*/*\<alarm ID>
+
+- From DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41059-->: `https://<DMA IP>/monitoring/alarm/<DMA ID>/<element ID>/<root alarm ID>/<alarm ID>`
+
+- Prior to DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12: `http(s)://<DMA IP>/monitoring/alarm/<DMA ID>/<alarm ID>`
```

Comment:
The old path was: IP/monitoring/alarm/DMA ID/ROOT ALARM ID/ALARM ID. For both the old & new path, the last alarm id is optional

Reply:
I've made some changes based on your feedback in [commit 3994b8a](https://github.com/SkylineCommunications/dataminer-docs/pull/3898/commits/3994b8a6645e767ce653f1dd91fe6f331cba10f7). What do you think, @WoutMH?

Reply:
LGTM